api/controllers/emailSubscribe.js

```diff
@@ -164,8 +161,8 @@ exports.unProtectedPut = async function (args, res, next) {
   var Project = mongoose.model('Project');
 
   // find the object ID based on the email address
-  await EmailSubscribe.findOne({ email: emailAddress }, null, function (err, entity) {
-    defaultLog.info('Err', err);
+  await EmailSubscribe.findOne({ _schemaName: 'EmailSubscribe', email: emailAddress }, null, function (err, entity) {
```

Comment:
was this the problem, that there was no schemaName defined?

Reply:
This is what was causing the issue. Long story but `EmailSubscribe` is set up on line 160 with: 
```
var EmailSubscribe = mongoose.model('EmailSubscribe');
```
I learned the hard way that if you don't pass an actual defined schema as the second argument, `EmailSubscribe.findOne()` will search for documents of *any* model that contain an email field with that value.